app/models/desafio.js

```diff
@@ -29,7 +29,10 @@ export default Model.extend({
   escena: attr('string'),
   hasAutomaticGrading: attr('boolean', { defaultValue: true }),
   estiloToolbox: attr('string', { defaultValue: "desplegable" }),
-  grupo: belongsTo('grupo'),
+  grupo: belongsTo('grupo', {async: false}), /* {async: false} is needed because the custom challenges should not have a group.
+  Without this, this.grupo is an empty ember object, instead of undefined. We want this to be undefined so we dont have to refactor other places
+  of the app to consider the case in which the group is an empty ember object instead of undefined.
+  */
```

Comment:
tl;dr: Sin el `{async: false}` `this.grupo` existe pero no tiene nada, asi que todos los `grupo.get('cualquier cosa')` explotan.

[Acá abajo de todo](https://api.emberjs.com/ember-data/release/functions/@ember-data%2Fmodel/belongsTo/) se explica en la documentación de ember. Como nosotros cargamos localmente los grupos al mismo tiempo que los desafíos, creería que esto no _debería_ traer problemas.

Esto explotaba acá en el `challengesInTheSameGroup` y en:
https://github.com/Program-AR/pilas-bloques/blob/fdb94258ff8c7f0582d81fb4115b72e1c1961527/app/services/challenge-expectations.js#L74-L83